vm/vm.go

```diff
@@ -246,7 +247,14 @@ func (v *vm) Execute(txns []core.Transaction, declaredClasses []core.Class, bloc
 		return nil, nil, errors.New(context.err)
 	}
 
-	return context.actualFees, context.traces, nil
+	traces := make([]TransactionTrace, len(context.traces))
+	for index, traceJSON := range context.traces {
+		if err := json.Unmarshal(traceJSON, &traces[index]); err != nil {
+			return nil, nil, fmt.Errorf("unmarshal trace: %v", err)
```

Comment:
I failed to point this out before merging, but this error should be mapped to `ErrInternal` on the RPC side.